### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by Rantelo**

### CODE REVIEW

File: src/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      to: "https://github.com/Intelimetrica/GUIM",
+      permission: "default"
+    },
+    {
+      name: "Intelimetrica",
+      to: "http://intelimetrica.com",
+      permission: "default"
+    }
+  ]
+}
```
Comment (ChrisAldama) [inline src/Navbar/Navbar.js]: semicolon is missing

File: src/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  }
+
+  _buildHeaders() {
+    const { modules } = this.props;
+    if(modules.length < 1){
+      return;
+    }
+    return modules.filter((module) => this._havePermission(this.state.permissions, module.permission))
+      .map((module) => {
+        if(module.mobile === true) return;
```
Comment (ChrisAldama) [inline src/Navbar/Navbar.js]: you can skip this line if you use Array.filter and then map
.filter(module => !module.mobile)
.map(module => {
});

File: src/Navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        {
+          name: "Intelimetrica",
+          to: "http://intelimetrica.com",
+          permission: "default",
+          mobile: true
+        }
+      ]
+    }
+  ]
+}
```
Comment (ChrisAldama) [inline src/Navbar/Navbar.js]: missing semi colon

File: src/Navbar/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+const themes = {
+  dark: "GUIMNavbarDark",
+  light: "GUIMNavbarLight"
+}
```
Comment (ChrisAldama) [inline src/Navbar/Navbar.js]: missing semicolon

File: src/Navbar/Navbar.js (hunk 2 above)
Comment (Rantelo) [reply src/Navbar/Navbar.js]: True, looks cleaner this way.

**Pull request merged**